pkg/tuned/controller.go

```diff
@@ -87,6 +92,16 @@ const (
 	ocpTunedImageEnv           = ocpTunedHome + "/image.env"
 	tunedProfilesDirCustomHost = ocpTunedHome + "/profiles"
 	tunedRecommendDirHost      = ocpTunedHome + "/recommend.d"
+
+	tunedDeferredUpdateStatusTrigger = ocpTunedRunDir + "/statusctl"
```

Comment:
Why do we need this control file?  Cannot a status update by enqued by enqueing a Change?

Reply:
It's an alternative I'm willing to explore once the e2e tests and the core flows are fully stabilized

Reply:
@jmencak so `energy_perf_bias` is not usable on CI. Is there another simple change I can use on my test profiles?

Reply:
regarding the proposed removal of `statusctl`, LGTM and e2e tests pass locally, so merging in the current code.
Next, need to fix e2e tests in CI.